driver-core/src/main/com/mongodb/internal/operation/BatchCursor.java

```diff
@@ -98,4 +104,9 @@ public interface BatchCursor<T> extends Iterator<List<T>>, Closeable {
     ServerCursor getServerCursor();
 
     ServerAddress getServerAddress();
+
+    default List<List<T>> exhaustCursor() {
```

Comment:
Given that `exhaustCursor` is a method on `AsyncBatchCursor`, we don't need to clarify that it applies to a cursor:

```suggestion
    default List<List<T>> exhaust() {
```